src/components/backend-ai-agent-summary-list.ts

```diff
@@ -72,6 +72,12 @@ export default class BackendAIAgentSummaryList extends BackendAIPage {
       IronFlexAlignment,
       // language=CSS
       css`
+        vaadin-grid {
+          border: 0;
+          font-size: 14px;
+          height: calc(100vh - 229px);
```

Comment:
```suggestion
          height: calc(100vh - 182px);
```
The height of the agent-summary page is slightly different.

Agent Summary page
![image](https://user-images.githubusercontent.com/28584164/224624605-898b91c1-fa30-4078-9538-18d1560f9a90.png)

Other components
![image](https://user-images.githubusercontent.com/28584164/224624664-163bead5-afcb-49fa-8925-483c64a7636a.png)